StringOps.java

```diff
@@ -22,21 +22,133 @@ public class StringOps {
     //////                                               ///////
     ////////////////////////////////////////////////////////////
     public static void main(String[] args) {
-        
+    String s="Hello World";
+    //String s="One two tHRee world";
+    //System.out.println(capVowelsLowRest(s));        
+    System.out.println(camelCase (s));
+    //String string="Hello world";
+    //char chr='l';
+    //System.out.println(allIndexOf (string, chr));
+
     }
 
-    public static String capVowelsLowRest (String string) {
+    public static String capVowelsLowRest (String string) { //why doesnt it work when the first char is an uppercase vowel?
         // Write your code here:
-        return "";
+        String answer="";
+        for(int i=0;i<string.length(); i++)
+        {
+            if((string.charAt(i)=='a') || (string.charAt(i)=='e') || (string.charAt(i)=='i') || (string.charAt(i)=='o') || (string.charAt(i)=='u'))
```

Comment:
Too long if statements - you could simplify it like this:
```suggestion
        String answer="";
        String vowelsString = "aeiou";
        for(int i=0;i<string.length(); i++)
        {
            if(vowelsString.indexOf(string.charAt(i) != -1)
```

Reply:
Thank you :)
A great way of using strings, I'll remember that from now on.